lua/wikis/rainbowsix/Tier/Data.lua

```diff
@@ -105,5 +105,21 @@ return {
 			link = 'Showmatches',
 			category = 'Showmatch Tournaments',
 		},
+		points = {
+			value = 'Points',
+			sort = 'B1',
+			name = 'Points',
+			short = 'Points',
```

Comment:
Might be a better fit for e.g. tournaments ticker on mainpage
```suggestion
			short = 'Pts,',
```

Reply:
shouldn't it be `.` instead of `,`?

Reply:
yeah i can't type